File: apps/codemap-search/experiments/event-navigation-poc/language-corpus/fixtures/scala/callbacks.scala
class Router {
  var primary: () => Unit = () => ()
  var secondary: () => Unit = () => ()
  def setPrimary(cb: () => Unit): Unit = { this.primary = cb } // store_primary
  def setSecondary(cb: () => Unit): Unit = { this.secondary = cb } // store_secondary
  def firePrimary(): Unit = { primary() } // call_primary
  def fireSecondary(): Unit = { secondary() } // call_secondary
}
class Other {
  var primary: () => Unit = () => ()
  def firePrimary(): Unit = { primary() } // call_other
}

File: apps/codemap-search/experiments/event-navigation-poc/language-corpus/fixtures/kotlin/callbacks.kt
class Router {
    var primary: (() -> Unit)? = null
    var secondary: (() -> Unit)? = null
    fun setPrimary(cb: () -> Unit) { this.primary = cb } // store_primary
    fun setSecondary(cb: () -> Unit) { this.secondary = cb } // store_secondary
    fun firePrimary() { primary?.invoke() } // call_primary
    fun fireSecondary() { secondary?.invoke() } // call_secondary
}
class Other {
    var primary: (() -> Unit)? = null
    fun firePrimary() { primary?.invoke() } // call_other
}
